pages/account/statistics.js

```diff
@@ -155,7 +155,7 @@ export default function Statistics({ data, profile, progress }) {
         </div>
 
         <h1 className="text-4xl mb-4 font-bold">
-          Your Statistics for {profile.name} ({profile.username})
+          Your Statistics for {profile.name} (<a href={"https://linkfree.io/" + (profile.username)}>{profile.username}</a>)
```

Comment:
We need to use `Link` and not hardcode the url...

```suggestion
          Your Statistics for {profile.name} (
          <Link
            href={`${process.env.NEXT_PUBLIC_BASE_URL}/${profile.username}`}
          >
            {profile.username}
          </Link>
          )
```

Reply:
We will need to import `Link` from components, so I will merge into a temporary branch and fix